### PR TITLE
conflict when using picker popOver on a popOver contains the form

### PR DESCRIPTION
**Pull request by e3bmo3ty**

### CODE REVIEW

File: controllers/picker.js [hunk 1]
```diff
@@ -40,6 +41,7 @@ var picker = {
  * @param {Number} [args.picker.type=Ti.UI.PICKER_TYPE_DATE] On Android, if this is `Ti.UI.PICKER_TYPE_DATE` or `Ti.UI.PICKER_TYPE_TIME` this will trigger the related dialogs.
  * @param {String} [args.picker.valueFormat="YYYY-MM-DD"] Format in which the value is set and get.
  * @param {String} [args.picker.textFormat] Format in which the value is displayed (defaults to `valueFormat`).
+ * @param {Boolean} [args.disablePopover] a flag to disable popOver on iPad.
```
Comment (FokkeZB) [inline controllers/picker.js]: Change to: "Set to `true` to not use the popover for the picker iPad.

File: controllers/picker.js [hunk 2]
```diff
@@ -101,7 +105,7 @@ function focus(e) {
 
   if (OS_IOS) {
 
-    if (Ti.Platform.osname === 'ipad') {
+      if (Ti.Platform.osname === 'ipad' && !disablePopover) {
```
Comment (FokkeZB) [inline controllers/picker.js]: Remove the extra space before the `if`

**Comment by e3bmo3ty** [comment]
Done 👍

File: controllers/picker.js [hunk 3]
```diff
@@ -101,7 +105,7 @@ function focus(e) {
 
   if (OS_IOS) {
 
-    if (Ti.Platform.osname === 'ipad') {
+     if (Ti.Platform.osname === 'ipad' && !disablePopover) {
```
Comment (FokkeZB) [inline controllers/picker.js]: Still one space here, sorry :)

**Comment by e3bmo3ty** [comment]
NVM :) , Done.

**Comment by FokkeZB** [comment]
Thanks man!

**Pull request merged**